src/main/java/mops/rheinjug2/controllers/StudentController.java

```diff
@@ -83,14 +83,18 @@ public String getPersonal(final KeycloakAuthenticationToken token, final Model m
   public String reportSubmit(final KeycloakAuthenticationToken token, final Model model,
                              final Long eventId) {
     if (eventId == null) {
-      return "redirect:rheinjug2/student/visitedevents";
+      return "redirect:/rheinjug2/student/visitedevents";
     }
     final LocalDateTime today = LocalDateTime.now(ZoneId.of("Europe/Berlin"));
     final Account account = AccountCreator.createAccountFromPrincipal(token);
     final Event event = modelService.loadEventById(eventId);
     if (event == null) {
       throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Event ID not found!");
     }
+    final LocalDateTime deadline = modelService.getDeadline(account.getName(), event);
+    if (deadline.isBefore(LocalDateTime.now())) {
```

Comment:
try to put a ZoneId into LocalDateTime.now() in case the host machine has the wrong time zone